ephys.py

```diff
@@ -0,0 +1,113 @@
+import datajoint as dj
+
+import actions
+import equipment
+
+schema = dj.schema(dj.config['names.%s' % __name__], locals())
+
+
+@schema
+class BrainLocation(dj.Manual):
+    # XXX: not connected to reference.*
+    # <class 'electrophysiology.models.BaseBrainLocation'>
+    definition = """
+    ccf_ap:			float		# ccf ap
+    ccf_dv:			float		# ccf dv
+    ccf_lr:			float		# ccf lr
+    ---
+    allen_ontology:		varchar(255)	# allen ontology
+    json:			varchar(255)	# json
+    """
+
+
+@schema
+class ExtracellularRecording(dj.Manual):
+    # <class 'electrophysiology.models.ExtracellularRecording'>
+    # <class 'electrophysiology.models.ProbeInsertion'>
+    # <class 'electrophysiology.models.ProbeModel'>
+    definition = """
+    -> actions.Session
+    start_time:                 datetime        # start time
+    end_time:                   datetime        # end time
+    ---
+    -> equipment.Appliance.Amplifier
+    -> equipment.Appliance.DAQ
+    ground_electrode:		varchar(255)    # ground electrode
+    reference_electrode:	varchar(255)    # reference electrode
+    filter_info:		varchar(255)    # filter information
+    lfp:                        longblob        # low-pass filtered
+    impedance:                  longblob        # impedance
+    gains:                      longblob        # gain
+    """
+
+    class Probe(dj.Part):
+        # XXX: TODO: split by electrode x/y/z; tie to other data
+        definition = """
+        probe_id:               int             # probe id
+        ---
+        -> BrainLocation
+        -> equipment.Appliance.ProbeModel
+        site_positions:         blob            # probe site positions
+        channel_mapping:        blob            # channel mapping
+        """
+
+
+@schema
+class SpikeSorting(dj.Manual):
```

Comment:
Instead of naming a table after the algorithm/action, let's name it to describe the entity within the table. Here, I'd suggest renaming to something like `SortedUnitGroup`.

Reply:
done.